tests/test_trailedEdgeSources.py

```diff
@@ -0,0 +1,299 @@
+                self.exposure.mask.setMaskPlaneValues(planes['EDGE'], dim[0] - 20, dim[0] - 1, y)
+
+        totFlux = self.exposure.image.array.sum()
+        self.exposure.image.array /= totFlux
+        self.exposure.image.array *= trail.instFlux
+
+        record.set(self.keys["instFlux"], trail.instFlux)
+        self._installFootprint(record, self.exposure.getImage())
+
+        return record, self.exposure.getImage()
+
+
+# Following from test_trailedSources
+@classParameters(length=length, theta=angle, xc=x_coord, yc=y_coord)
+class TrailedEdgeSourcesTestCase(AlgorithmTestCase, lsst.utils.tests.TestCase):
+    """ Test if ext_trailedSources_Naive_flag_edge is set correctly.
+
+    Given a `TrailedSource`, test if the edge flag is set correctly in the
+    source catalog after the
+    'lsst.meas.extensions.trailedSources.Naive.Plugin.makeTrailedSourceMeasurementTask'
+    has been run on the source catalog.
+    """
+
+    def setUp(self):
+        self.center = Point2D(50.1, 49.8)
+        self.bbox = Box2I(lsst.geom.Point2I(-20, -30), Extent2I(140, 160))
+        self.dataset = TrailedTestDataset(self.bbox)
+
+        # Trail which extends into edge pixels
+        self.trail = TrailedEdgeSource(100000.0, self.length, self.theta, self.xc, self.yc)
+        self.dataset.addTrailedSource(self.trail)
+
+    def tearDown(self):
+        del self.center
+        del self.bbox
+        del self.trail
+        del self.dataset
+
+    def testEdgeFlag(self):
+        """Test if edge flags are correctly set in NaivePlugin.py
+
+        Given a `TrailedTestDataset`, run the NaivePlugin measurement and
+        check that the trailed sources have the edge flag set. [100,100] does
+        not contain any edge pixels and should not have a flag set, [20,20]
+        crosses into the edge region on onle one side and should have the edge
+        flag set, and [-20,-30] extends off the chip and should have the edge
+        flag set.
+        """
+        # Set up and run Naive measurement.
+        task = TrailedTaskSetup.makeTrailedSourceMeasurementTask(self,
+                                                                 plugin="ext_trailedSources_Naive",
+                                                                 dependencies=("base_SdssCentroid",
+                                                                               "base_SdssShape")
+                                                                 )
+        exposure, catalog = self.dataset.realize(5.0, task.schema, randomSeed=0)
+        task.run(catalog, exposure)
+        record = catalog[0]
+
+        # Check that x0, y0 or x1, y1 is flagged as an edge pixel
+        x1 = int(record['ext_trailedSources_Naive_x1'])
+        y1 = int(record['ext_trailedSources_Naive_y1'])
+        x0 = int(record['ext_trailedSources_Naive_x0'])
+        y0 = int(record['ext_trailedSources_Naive_y0'])
+
+        # Test Case with no edge pixels
+        if record['truth_x'] == 100:
+
```

Comment:
Suggest removing blank lines after the if / elif / else lines

Reply:
Removed